gcsfs/tests/test_core.py

```diff
@@ -1493,3 +1493,28 @@ def test_find_maxdepth(gcs):
 
     with pytest.raises(ValueError, match="maxdepth must be at least 1"):
         gcs.find(f"{TEST_BUCKET}/nested", maxdepth=0)
+
+
+def test_sign(gcs, monkeypatch):
+    file = TEST_BUCKET + "/test.jpg"
+    with gcs.open(file, "wb") as f:
+        f.write(b"This is a test string")
+    assert gcs.cat(file) == b"This is a test string"
+
+    # `sign` is creating a google Client on its own, it needs a realistically
+    # looking credentials file.
+    if not gcs.on_google:
```

Comment:
C: can someone run it on google?

Reply:
Not by using the CI fixture. It might make sense to provide a config option to run the tests "for real", not sure how best to do that